src/reducers/palettesReducer/palettesReducer.js

```diff
@@ -1,7 +1,11 @@
 export const palettesReducer = (state = [], action) => {
   switch (action.type) {
     case 'SET_PALETTES':
-      return action.palettes
+      let fetchedPalettes = [];
+      action.palettes.forEach(project => {
+        project.forEach(palette => fetchedPalettes.push(palette))
+      });
+      return fetchedPalettes
```

Comment:
I'm a little confused about what these changes are doing. Can we talk through this?

Reply:
We're fetching the palettes based on each project, so each fetch of palettes comes back as an array. The final result is an array of project palette arrays [[1,2],[3,4]]. So in the reducer I'm looping over each array and then looping each array to push in each palette individually so that all our palettes are in one flat array when they go into the store.